erpnext/shopping_cart/cart.py

```diff
@@ -307,6 +323,32 @@ def get_customer(user=None):
 
 		return customer
 
+def get_debtors_account(cart_settings):
+	payment_gateway_account_currency = \
+		frappe.get_doc("Payment Gateway Account", cart_settings.payment_gateway_account).currency
+	
+	account_name = _("Debtors ({0})".format(payment_gateway_account_currency))
+	
+	debtors_account = get_account("Receivable", "Asset", is_group=0, account_name=account_name)
```

Comment:
Check existing debtors account based on account_type and account_currency and company, not based on account_name. If not found create it.